### DISCUSSION TITLE
NootedRed AMD Ryzen 3500u

### DISCUSSION THREAD

**Discussion by Marquin099** [ChefKissInc/NootedRed#308] [Q&A] [answered]
Hello, how are you? I hope you are well!
So I wanted to know if you can help me? I have an AMD Ryzen 5 3500U Vega 8 20GB of RAM, but on MacOS Sonoma, Nootedred makes my system crash and restart every time. It only works if I use Whatavergreen, but if I use Nootedred on Ventura it works perfectly. On Sonoma and Sequoia it doesn't work, it crashes every time. Is there a way to solve this?

**Comment by nononymousse** [comment, marked as answer]
that issue is already discussed and worked on on #235

**Comment by Moony-JR-window** [comment]
I'm stuck at Sonoma , Sequoia 
- model : asus ryzen 5 3500u Ram 16GB 
- when boot start maybe 10 - 15 second it stuck machine 
- when to exit should click power to restart or off machine

**Comment by csatwood** [comment]
Plan approved. Initialized local repository at \`/Users/chris/Developer/repositories/flexoki-theme-templates/\` and pushed to [Going-into-the-Aether/flexoki-theme-templates](https://github.com/Going-into-the-Aether/flexoki-theme-templates).

Moving to Phase 1: Terminal preferences audit and repo file verification.

**Reply by VisualEhrmanntraut** [reply]
what?

**Comment by jaespinozasj** [comment]
I have the same problem. Is there any way to make it work on newer versions of Mac? It runs perfectly on Ventura, but if I update to a newer version, it freezes. I've removed the dynamic background and user settings, but it still freezes, for example, when I open the menu to restart.
